schema/stripe-app-local.schema.json

```diff
@@ -0,0 +1,53 @@
+{
+  "$id": "https://stripe.com/stripe-app-local.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "title": "Stripe App local development manifest",
+  "description": "A manifest describing a Stripe App.",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "extends": {
```

Comment:
I think this is the right call to not make extends required because devs might just make a local copy of the main manifest and make changes there.